README.md

```diff
@@ -1192,6 +1194,32 @@ targets:
   - name: hex
 ```
 
+### Git Repository (`commit-on-git-repository`)
+
+Takes a tarball and pushes the contents to a git repository.
```

Comment:
```suggestion
Takes a tarball and pushes the unpacked contents to a git repository.
```

Not 100% sure, but maybe this is more explicit...?

Reply:
Makes sense to me!